vignettes/v02_bivariate.Rmd

```diff
@@ -91,11 +108,28 @@ WoJ %>%
 
 Additional options include:
 
-- `var.equal`: By default, `t_test()` will assume equal variances for both groups. Set `var.equal = FALSE` to compute t-Tests with the Welch approximation to the degrees of freedom.
 - `pooled_sd`: By default, the pooled variance will be used the compute Cohen's $d$ effect size estimates ($s = \sqrt\frac{(n_1 - 1)s^2_1 + (n_2 - 1)s^2_2}{n_1 + n_2 - 2}$).
 Set `pooled_sd = FALSE` to use the simple variance estimation instead ($s = \sqrt\frac{(s^2_1 + s^2_2)}{2}$).
 - `paired`: Set `paired = TRUE` to compute a paired t-Test instead. It is advisable to specify the case-identifying variable with `case_var` when computing paired t-Tests, as this will make sure that data are properly sorted.
 
+Previously, the (now deprecated) option of `var.equal` was also available. This has been overthrown, however, as `t_test()` now by default tests for equal variance (using a Levene test) to decide whether to use pooled variance or to use the Welch approximation to the degrees of freedom.
+
+`t_test()` also provides a one-sample t-Test if you provide a `mu` argument:
+
+```{r}
+WoJ %>% 
+  t_test(autonomy_emphasis, mu = 3.9)
+```
+
+Of course, also the result from t-Tests can be visualized easily as such:
+
+```{r}
+WoJ %>% 
+  t_test(temp_contract, autonomy_selection, autonomy_emphasis) %>% 
+  visualize()
+```
+
+
 ## Compute one-way ANOVAs
 
 `unianova()` will compute one-way ANOVAs for one group variable and one or more test variables. Output includes test statistics and $\eta^2$ effect size estimates.
```

Comment:
And omega_squared, if Welch's approximation is used to account for unequal variances.